### PR TITLE
Printing docs update

### PR DESCRIPTION
**Pull request by jxlin123**
I've updated the documentation to reflect the newer printing system in place. Let me know if there is something I should add or change!

Addresses https://github.com/ocf/ocfweb/issues/529

Issues page for the printing system itself: https://github.com/ocf/puppet/issues/731

### CODE REVIEW

**Comment by jxlin123** [comment]
@fydai sounds good. If no one else has any comments or suggested changes, then I guess merge...

File: ocfweb/docs/docs/staff/backend/printhost.md [hunk 1]
```diff
@@ -1,20 +1,29 @@
...
 nicknamed enforcer after one of the scripts, plugs into CUPS as a hook that
 looks at jobs before and after going to the printer. It records jobs in a
 database that keeps track of how many pages each user has printed, rejecting
-jobs that go over quota. The high level flow of data through the print system
+jobs that go over quota. The high level flow of data through printhost
 looks like this:
 
 ```
-   [Application]
+  [Local machine]
```
Comment (cg505) [inline ocfweb/docs/docs/staff/backend/printhost.md]: should we add the local machine CUPS to this chart as well?

File: ocfweb/docs/docs/staff/backend/printhost.md [hunk 2]
```diff
@@ -1,20 +1,29 @@
...
+Print jobs at the OCF are processed both locally and by our central printing
+server, known as printhost or whiteout. All local machines and printhost use
+[CUPS][cups], an open source UNIX printing system, to manage printing jobs.
+
+A CUPS instance runs on each of our desktops, as well as on our public and staff
+login servers (tsunami and supernova, respectively). All print jobs, after going
+through a filter that rasterizes any PDFs, are then forwarded onto printhost
+for further processing before finally being printed.
+
+The OCF's central print server is based around two components: CUPS,
```
Comment (cg505) [inline ocfweb/docs/docs/staff/backend/printhost.md]: You might want to reflow this paragraph. /shrug

File: ocfweb/docs/docs/staff/backend/printhost.md [hunk 3]
```diff
@@ -115,8 +141,7 @@ afford the toner.
 
 ## Print accounting
 
-The OCF uses a virtual CUPS printer backend called [Tea4CUPS][Tea4CUPS] to
-install a page accounting hook that runs before and after each job is actually
+The OCF uses a Tea4CUPS backend to install a page accounting hook that runs before and after each job is actually
```
Comment (cg505) [inline ocfweb/docs/docs/staff/backend/printhost.md]: wrap at 80 if possible

Comment (cg505) [reply ocfweb/docs/docs/staff/backend/printhost.md]: also you should specify that this happens on printhost not locally here

**Comment by jxlin123** [comment]
Cool, cool, and thanks for the comments so far!

Given all the recent comments and concerns about printing (see Slack -- the issues page is very out of date), I'm going to put this on the back burner for now, while we address the printing situation...

**Pull request closed without merging**